src/TestFramework/Factory.php

```diff
@@ -112,10 +116,23 @@ public function create(string $adapterName, bool $skipCoverage): AbstractTestFra
             );
         }
 
+        if ($adapterName === TestFrameworkTypes::CODECEPTION) {
+            $codeceptionConfigPath = $this->configLocator->locate(TestFrameworkTypes::CODECEPTION);
+            $codeceptionConfigContent = file_get_contents($codeceptionConfigPath);
+
+            return new CodeceptionAdapter(
+                new TestFrameworkExecutableFinder(CodeceptionAdapter::EXECUTABLE),
```

Comment:
Not Found For Me. Changed It To `TestFrameworkFinder` and started working again.